sc-tools/sc-builder/src/gwf_parser.cpp

```diff
@@ -0,0 +1,171 @@
+#include "gwf_parser.hpp"
+
+std::string GwfParser::XmlCharToString(std::unique_ptr<xmlChar, XmlCharDeleter> const & ptr)
+{
+  return ptr ? std::string(reinterpret_cast<char const *>(ptr.get())) : "";
```

Comment:
Are there any ways to convert xmlChar array to std::string? Using cast in this code can be dangerous